variant-remapping-automation/prepare_remapping_table.py

```diff
@@ -0,0 +1,246 @@
+        scientific_name = get_scientific_name(tax, scientific_names_from_release_table)
+        if tax in taxonomy_with_mismatch_assembly:
+            # entry already made in insert_entries_for_new_assembly_found_in_ensembl
+            continue
+        for asm in assemblies:
+            if asm == tax_latest_support_asm[tax]['assembly'] or asm not in asm_studies_acc_after_remapping:
+                continue
+            # find out if there are new studies (study4) in assembly (asm5) which is alreday remapped to asm6
+            # asm5 (study4) -> asm6
+            sources = get_assembly_sources(asm, eva_assemblies, dbsnp_assemblies)
+            insert_entry_into_db(private_config_xml_file, sources, tax, scientific_name, asm,
+                                 tax_latest_support_asm[tax]['assembly'], remapping_version,
+                                 release_version, len(asm_studies_acc_after_remapping[asm]), -1,
+                                 "'{" + "".join(asm_studies_acc_after_remapping[asm]) + "}'")
+
+
+def insert_entry_into_db(private_config_xml_file, sources, tax, scientific_name, org_asm, target_asm, remapping_version,
+                         release_version, num_studies, num_ss_ids, study_accessions):
+    with get_metadata_connection_handle('production_processing', private_config_xml_file) as pg_conn:
+        query = f"insert into eva_progress_tracker.remapping_tracker (source, taxonomy, scientific_name, " \
+                f"origin_assembly_accession, assembly_accession, remapping_version, release_version, " \
+                f"num_studies, num_ss_ids, study_accessions) " \
+                f"values ('source_placeholder',{tax},'{scientific_name}','{org_asm}','{target_asm}'," \
+                f"'{remapping_version}', {release_version}, " \
+                f"{num_studies}, {num_ss_ids}, {study_accessions}) "
+        if 'EVA' in sources:
+            query_eva = query.replace('source_placeholder', 'EVA')
+            logger.info(f"{query_eva}")
+            execute_query(pg_conn, query_eva)
+        if 'DBSNP' in sources:
+            query_dbsnp = query.replace('source_placeholder', 'DBSNP')
+            logger.info(f"{query_dbsnp}")
+            execute_query(pg_conn, query_dbsnp)
+
+
+def get_tax_all_supported_assembly_from_eva(private_config_xml_file):
+    with get_metadata_connection_handle('production_processing', private_config_xml_file) as pg_conn:
+        query = f"""select taxonomy_id, assembly_id from evapro.supported_assembly_tracker"""
+
+        tax_asm_list = {}
+        for tax, asm in get_all_results_for_query(pg_conn, query):
+            if tax not in tax_asm_list:
+                tax_asm_list[tax] = {asm}
+            else:
+                tax_asm_list[tax].add(asm)
+
+        return tax_asm_list
+
+
+def get_assembly_latest_remapping_time(private_config_xml_file):
+    asm_remaptime = {}
+    # TODO: change to production_processing after filling data
```

Comment:
Can you just use `remapping_report_time` for the previous release version in the remapping tracking table for this, rather than checking the job tracker?

Reply:
As discussed, job tracker looks more accurate with dates, so using that for now